README.md

```diff
@@ -1,19 +1,86 @@
+<img src="https://user-images.githubusercontent.com/3044853/65060781-6d032d00-d970-11e9-9bb2-44c1811f80b5.png" height="100px" align="left"/>
+
 # Prodo
 
-Prodo is a full-stack framework to build apps faster.
+[![CircleCI](https://circleci.com/gh/prodo-ai/prodo.svg?style=svg)](https://circleci.com/gh/prodo-ai/prodo) 
+[![npm version](https://badge.fury.io/js/%40prodo%2Fcore.svg)](https://badge.fury.io/js/%40prodo%2Fcore)
+
+Prodo is a React framework to write performant and scalable web
+apps with as little boilerplate as possible.
+
+[Documentation](https://prodo-docs.web.app)
+
+## Key benefits
+
+- Drastically simplified state management
+- Absolute minimal boilerplate, especially compared to Redux
+- Blazingly fast re-rendering, similar to MobX
```

Comment:
Technically this isn't true - we just re-render less than MobX because our updates are batched.